src/components/crossDevice/SwitchDevice/style.css

```diff
@@ -0,0 +1,54 @@
+.container {
+  height: 96px;
```

Comment:
What did we conclude on pixels vs rem? I wasn't part of the debate, but you seem to be using them interchangeably

Reply:
There is a ticket for that in Jira https://jira.onfido.co.uk/browse/CX-1217
We haven't started the debate yet. We can use both rem and px at the same time, it's not a problem, as long as we do it properly so that the UI looks good when a user changes the front size in their browser settings. I wonder what's the percentage of users that change the font size in their browser settings, and if it's big enough maybe we should tackle this issue as part of the accessibility epic, whenever we will work on that.